nmdc_schema/migrators/migrator_from_X_to_PR19_and_PR70.py

```diff
@@ -0,0 +1,101 @@
+from nmdc_schema.migrators.migrator_base import MigratorBase
+from nmdc_schema.migrators.helpers import load_yaml_asset
+import difflib
```

Comment:
Note to self: `difflib` is included in the Python standard library. Docs: https://docs.python.org/3/library/difflib.html